src/main/resources/META-INF/gradle-plugins/org.jruyi.thrift.properties

```diff
@@ -1,16 +0,0 @@
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-implementation-class=org.jruyi.gradle.thrift.plugin.ThriftPlugin
```

Comment:
Q: Don't we need to migrate this file for the new package?

Reply:
Maybe we don't need it anymore, I tried to check the latest document of publishing and it doesn't talk about this file.